### PR TITLE
Changes from dry run

### LINKED ISSUE #14: Remove some slides and exercises
During the dry run of this course, we realised that we have too much content for the time. We identified some parts that can be removed, and others that can be skipped if needed.

Remove:

- License part (or remove the last two slides, and skip rest if needed)
- fstring slide
- config file slide (could stay in and be skipped if needed?)

There are also some smaller changes, as removing or adding incremental showing of bullet points.

### PR DESCRIPTION
**Pull request by MarionBWeinzierl**
Closes #14 . Implement some of the changes from the dry run (see issue)

### CODE REVIEW

File: slides/python.qmd [hunk 1]
```diff
@@ -245,21 +244,21 @@ Get in touch:
...
-\
-
 {{< fa pencil >}} \ Marion Weinzierl
 
 {{< fa solid envelope >}} \ [mw925[AT]cam.ac.uk](mailto:mw925@cam.ac.uk)
 
 {{< fa brands github >}} \ [MarionBWeinzierl](https://github.com/MarionBWeinzierl)
 
 {{< fa brands mastodon >}} \ [\@MarionBWeinzierl\@mast.hpc.social](https://mast.hpc.social/@MarionBWeinzierl)
+
```
Comment (MarionBWeinzierl) [inline slides/python.qmd:254]: This change is just to that the full contact info for everyone is visible and not hidden by the logo

File: slides/_comments_docstrings.qmd [hunk 2]
```diff
@@ -180,84 +176,6 @@ def calculate_gyroradius(mass, v_perp, charge, B, gamma=None):
 ::::
```
Comment (jatkinson1000) [inline slides/_comments_docstrings.qmd:179]: I feel like commenting out with a `<!-- -->` could be preferable so the content is not 'lost' as you do below.

**Review by jatkinson1000 — COMMENTED** [review]
Thanks @MarionBWeinzierl a comment in there.

Also, I forget the exact discussion, but on reflection here I would definitely cut the 'config file' part as this is just TMI at the end of a packed session and never lands very well.
Leaving the slide in to be visible is an option, but I am certain I would not cover it.
If anything I think fstrings would be easier for people to grasp if we somehow find ourselves with extra time.

File: slides/_comments_docstrings.qmd (hunk 2 above)
Comment (MarionBWeinzierl) [reply slides/_comments_docstrings.qmd:179]: OK, happy to do that -- I thought I'd remove this one fully, as it it deprecated and therefore will probably not be taught in future

**Comment by MarionBWeinzierl** [comment]
> Thanks @MarionBWeinzierl a comment in there.
> 
> Also, I forget the exact discussion, but on reflection here I would definitely cut the 'config file' part as this is just TMI at the end of a packed session and never lands very well. Leaving the slide in to be visible is an option, but I am certain I would not cover it. If anything I think fstrings would be easier for people to grasp if we somehow find ourselves with extra time.

Any preferences about leaving the slides in and skipping them, or commenting them out for the case you have time?

File: slides/_comments_docstrings.qmd (hunk 2 above)
Comment (MarionBWeinzierl) [reply slides/_comments_docstrings.qmd:179]: done

**Comment by MarionBWeinzierl** [comment]
> > Thanks @MarionBWeinzierl a comment in there.
> > Also, I forget the exact discussion, but on reflection here I would definitely cut the 'config file' part as this is just TMI at the end of a packed session and never lands very well. Leaving the slide in to be visible is an option, but I am certain I would not cover it. If anything I think fstrings would be easier for people to grasp if we somehow find ourselves with extra time.
> 
> Any preferences about leaving the slides in and skipping them, or commenting them out for the case you have time?

For the time being I have removed both. However, not sure whether the title of the section is then still fitting (maybe with the "honorable mentions" it is), or whether it should be changed. Or another slide added to mention the stuff that we removed.

**Review by jatkinson1000 — APPROVED** [review]
Punch it Chewie.

@MarionBWeinzierl I can't see we will have lots of spare time, so both removed is probably fine. We can always mention or demo if we feel the need to.

At this point let's get something deliverable for the summer school. We can refine and add back at a later date after next week if so desired.

**Pull request merged**